android/brave_java_resources.gni

```diff
@@ -898,3 +898,7 @@ brave_java_resources = [
   "java/res/xml/unstoppable_domains_preferences.xml",
   "java/res/xml/use_custom_tabs_brave_preference.xml",
 ]
+
+# XML preprocessing
+brave_java_preprocess_xml_sources = []
```

Comment:
Why are these empty? Is it just a template for future usage?

Reply:
Yes, I want to separate the PR for the template and for actual application.